src/Xamarin.Android.Build.Tasks/Tasks/GetJavaPlatformJar.cs

```diff
@@ -69,7 +69,8 @@ public override bool Execute ()
 			}
 
 			platform            = GetTargetSdkVersion (platform, target_sdk);
-			JavaPlatformJarPath = Path.Combine (MonoAndroidHelper.AndroidSdk.TryGetPlatformDirectoryFromApiLevel (platform, MonoAndroidHelper.SupportedVersions), "android.jar");
+			var platformPath = MonoAndroidHelper.AndroidSdk.TryGetPlatformDirectoryFromApiLevel (platform, MonoAndroidHelper.SupportedVersions);
+			JavaPlatformJarPath = Path.Combine (platformPath ?? MonoAndroidHelper.AndroidSdk.GetPlatformDirectoryFromId (platform), "android.jar");
```

Comment:
I think it would make more sense to instead emit an error if `TryGetPlatformDirectoryFromApiLevel()` fails, especially considering that [`TryGetPlatformDirectoryFromApiLevel()` calls `GetPlatformDirectoryFromId()`](https://github.com/xamarin/xamarin-android-tools/blob/6ed824b/src/Xamarin.Android.Tools.AndroidSdk/AndroidSdkInfo.cs#L86).

If `null` is returned, it's because the directory doesn't exist.

Thus, I think we should instead have:

```csharp
static string TryGetAndroidJarPath (TaskLoggingHelper log, string platform)
{
    var platformPath = MonoAndroidHelper.AndroidSdk.TryGetPlatformDirectoryFromApiLevel (platform, MonoAndroidHelper.SupportedVersions);
    if (platformPath == null) {
        log.LogError ("XA1234", $"Could not determine Android SDK directory for {platform}.");
        return null;
    }
    return Path.Combine (platformPath, "android.jar");
}
```

The one downside here is that we don't see the directories that we're trying. I'm not sure that's a problem.